### ISSUE TITLE
Bug: VSCode install script doesn't get all dependencies

### ISSUE THREAD

**Issue by int19h** [WhitewaterFoundry/WLinux#24]
After running:
```sh
sudo /opt/installcode.sh
```
as per instructions on the main page, I see that VSCode is installed, and the `code` binary is there. But when run:
```
int19h@ANGBAND ~> code
/usr/share/code/bin/../code: error while loading shared libraries: libXss.so.1: cannot open shared object file: No such file or directory
``` 
After getting past that, it's also missing libasound2.

**Comment by int19h** [comment]
To manually install the missing deps:
```sh
apt install libxss1 libasound2
```

**Comment by patrick330602** [comment]
it is a easy fix. Thanks for the feedback.

**Comment by sirredbeard** [comment]
This was a bug in 1.1.5, the first store release. It has been patched in 1.1.15 which is pending release now.
